lua/entities/gmod_wire_plug.lua

```diff
@@ -62,7 +62,7 @@ end
 function ENT:Setup( ArrayInput )
 	self.ArrayInput = ArrayInput or false
 
-	if (!self.Inputs or !self.Outputs or self.ArrayInput != old) then
+	if not (self.Inputs and self.Outputs) then
```

Comment:
Why did you erase this check? The check for ArrayInput is necessary or  else the entity won't be updated properly if you left click it with the tool.

Reply:
IDK how but couldn't save the file and commit that.